docs/developers/new_dataset_to_armory.md

```diff
@@ -0,0 +1,120 @@
+# How to add a new dataset into armory
+I will use the RAIT dataset as an example. It is an object detection dataset.
```

Comment:
Should read
> It is an object detection dataset drawn from TwoSix field exercises under the RAITE program. This dataset will not be available to you.  The characteristics of this dataset are:

**Chris we should strongly consider making this accessible at least upon request. We'll have to ask Etienne about distribution limitations that it might have**